SU-SWS/stanford_news
# D8CORE-2055: fix to the mobile news list

# READY FOR REVIEW

# Summary
- Fixed the spacing to large for the news list
- Removed the media when lg-xs

# Needed By (Date)
- ASAP

# Urgency
- high

# Steps to Test

1. pull in this branch
2. look at the list page and see that the images are there for XL and gone for the XS-LG. 
3. Verify the padding is right for LG to XS between the menu and content

# Affected Projects or Products
- News

# Associated Issues and/or People
- D8CORE-2055

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
